superdesk/locators/__init__.py

```diff
@@ -23,6 +25,9 @@ def _load_json(file_path):
         return json.load(f)
 
 
-_dir_name = os.path.dirname(os.path.realpath(__file__))
-_locators_file_path = os.path.join(_dir_name, 'data', 'locators.json')
-locators = _load_json(_locators_file_path)
+def init_app(app):
+    app.locators = LocatorIndex()
+    superdesk.blueprint(bp)
+    _dir_name = os.path.dirname(os.path.realpath(__file__))
+    _locators_file_path = os.path.join(_dir_name, 'data', 'locators.json')
```

Comment:
refactoring looks good, but reading the title imo this is the part that should be configurable